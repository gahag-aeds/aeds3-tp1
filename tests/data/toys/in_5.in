6
0.0000 0.22848 0.38067 0.91648 0.73719 0.87217 
0.77152 0.0000 0.86833 0.75517 0.59657 0.50993 
0.61933 0.13167 0.0000 0.48913 0.64397 0.13118 
0.08352 0.24483 0.51087 0.0000 0.00992 0.20115 
0.26281 0.40343 0.35603 0.99008 0.0000 0.93794 
0.12783 0.49007 0.86882 0.79885 0.06206 0.0000 
