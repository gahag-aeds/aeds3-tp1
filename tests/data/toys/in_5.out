0.224402 0.372158 0.092104 0.024883 0.185067 0.101386 